src/main/java/com/openlattice/chronicle/api/ChronicleApi.java

```diff
@@ -160,6 +160,17 @@ void sendMessages(
             @Body List<MessageDetails> messageDetailsList
     );
 
+    /**
+     * Update staus for messages sent to partipants.
+     * @param organizationId - Id of the organization to which study belongs
+     */
+    @POST( BASE + ORGANIZATION_ID_PATH + MESSAGE_PATH + STATUS_PATH)
+    void updateMessageStatus(
+            @Path( ORGANIZATION_ID ) UUID organizationId,
+            @Query( MESSAGE_ID ) String messageId,
+            @Query( MESSAGE_STATUS ) MessageStatus messageStatus
+    ) throws ExecutionException;
```

Comment:
No need for the `throws`.

```suggestion
    )
```